docs/HISTORY.txt

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+7.1 - (2023-11-22)
```

Comment:
@razvanMiu Please bump the version to 8.0 and state here that this version removes RestAPI endpoints:
- @map-visualization
- @tableau-visualization

in favor of getContent